ui/perfherder/alerts/AlertTableRow.jsx

```diff
@@ -193,6 +201,51 @@ export default class AlertTableRow extends React.Component {
     );
   };
 
+  showTags = (id, tags) => {
```

Comment:
The `id` param sole purpose is to make it easier for testing this. But it's already very easy to test a badge by selecting it using `getByText`. That way, we won't need to provide the alert id as param.